pkg/kubectl/describe.go

```diff
@@ -53,7 +53,13 @@ import (
 // if the output could not be generated. Implementers typically
 // abstract the retrieval of the named object from a remote server.
 type Describer interface {
-	Describe(namespace, name string) (output string, err error)
+	Describe(namespace, name string, describerSetting *DescriberSetting) (output string, err error)
```

Comment:
At this point, no. Later, DescriberSetting could contain non-pointer variables you would not want to copy.

Any reason for it not to be a pointer?